integration_tests/integration/home.spec.ts

```diff
@@ -11,29 +11,29 @@ context('Home page', () => {
   })
 
   it('should see the feedback banner', () => {
-    cy.visit(homepageUrl.root)
-    const homepage: Homepage = Page.verifyOnPage(Homepage)
-    homepage.feedbackBanner().should('exist')
+    cy.visit(adjudicationUrls.homepage.root)
+    const homepageMockup: HomepagePage = Page.verifyOnPage(HomepagePage)
+    homepageMockup.feedbackBanner().should('exist')
   })
 
   it('should only see some tiles without the reviewer role', () => {
-    cy.visit(homepageUrl.root)
-    const homepage: Homepage = Page.verifyOnPage(Homepage)
-    homepage.feedbackBanner().should('exist')
-    homepage.startANewReportLink().should('exist')
-    homepage.continueAReportLink().should('exist')
-    homepage.viewYourCompletedReportsLink().should('exist')
-    homepage.viewAllCompletedReportsLink().should('not.exist')
+    cy.visit(adjudicationUrls.homepage.root)
+    const homepageMockup: HomepagePage = Page.verifyOnPage(HomepagePage)
```

Comment:
Not that its a big deal - but did you need to change the name from homepage to homepagemock? I'm surprised

Reply:
yep good spot, this was done before the bulk of the other changes and now it's unnecessary